### PR TITLE
Small Fix

### PR DESCRIPTION
**Pull request by sunhwapark**
Spell Doll > Shaman Doll

### CODE REVIEW

**Comment by Sunaries** [comment]
Good to merge. :+1:

File: EnglishTranslation-master/ITEM.tsv [hunk 1]
```diff
@@ -7070,7 +7070,7 @@ ITEM_20151001_007069	Merchant's Records Piece
 ITEM_20151001_007070	A paper that is likely to contain the records of the Kedorlaomer Merchants. You cannot see clearly what is written on it since monsters used to have it.
 ITEM_20151001_007071	Trap
 ITEM_20151001_007072	A trap for capturing monsters. As soon as it's placed, it'll start to attract the attention of monsters.
-ITEM_20151001_007073	Fresh Blood
+ITEM_20151001_007073	Body Fluid
```
Comment (Sunaries) [inline EnglishTranslation-master/ITEM.tsv]: Fresh Body Fluid

File: EnglishTranslation-master/QUEST_UNUSED.tsv [hunk 2]
```diff
@@ -887,50 +887,47 @@ QUEST_UNUSED_20151022_000886	Ah... a human in this place...{nl}There are demons
...
-QUEST_UNUSED_20151022_000903	We, Kedoran Merchant Alliance act harsh on the newcomers.{nl}Especially, Leopoldas is really scary.. He wants me to do this alone.
-QUEST_UNUSED_20151022_000904	If that's so, can you put this notice board on Krive Highway?{nl}Edita, our leader, is excavating this sacred object.{nl}
+QUEST_UNUSED_20151022_000902	We are in a trouble.{nl}We have to put up a notice board, but we're just too busy.{nl}
+QUEST_UNUSED_20151022_000903	The Kedoran Merchant Alliance act harsh on newcomers.{nl}Especially Leopoldas, he's really scary... He wants me to do this alone.
 QUEST_UNUSED_20151022_000905	I am counting on you to do this secretly without being detected by Leopoldas.{nl}I have to go somewhere.
 QUEST_UNUSED_20151022_000906	I am counting on you to do this secretly without being detected by Leopoldas.{nl}For sure.
-QUEST_UNUSED_20151022_000907	I am helping the Revelators who just started their journies from Kedoran Merchant Alliance.{nl}If you are okay, can you help us?
+QUEST_UNUSED_20151022_000907	I am helping the Revelators who have just started their journies from the Kedoran Merchant Alliance.{nl}If you don't mind, can you help us?
```
Comment (Sunaries) [inline EnglishTranslation-master/QUEST_UNUSED.tsv]: journies -> journeys

File: EnglishTranslation-master/QUEST_UNUSED.tsv [hunk 3]
```diff
@@ -887,50 +887,47 @@ QUEST_UNUSED_20151022_000886	Ah... a human in this place...{nl}There are demons
...
 QUEST_UNUSED_20151022_000908	It's not that difficult.{nl}You just have to pass the blessing which is needed for their journies.
```
Comment (Sunaries) [inline EnglishTranslation-master/QUEST_UNUSED.tsv]: journies -> journeys

File: EnglishTranslation-master/QUEST_UNUSED.tsv [hunk 4]
```diff
@@ -887,50 +887,47 @@ QUEST_UNUSED_20151022_000886	Ah... a human in this place...{nl}There are demons
...
 QUEST_UNUSED_20151022_000909	Thank you.{nl}Then, I will tell you how you can do it.{nl}
-QUEST_UNUSED_20151022_000910	Please look for the Revelator who just started their journies around Orsha and Klaipeda.{nl}When you this scroll to him, he will receive the blessing that is needed for the journey.
-QUEST_UNUSED_20151022_000911	Ah, and it won't be effective to the one who is already blessed so you please keep that in mind.
+QUEST_UNUSED_20151022_000910	Please look for the Revelators who have started their journies around Orsha and Klaipeda.{nl}When you use this scroll on them, they will receive the blessing that is needed for the journey.
```
Comment (Sunaries) [inline EnglishTranslation-master/QUEST_UNUSED.tsv]: journies -> journeys
`needed for the journey` -> `needed for their journeys`

File: EnglishTranslation-master/QUEST_UNUSED.tsv [hunk 5]
```diff
@@ -911,6 +913,7 @@ QUEST_UNUSED_20151022_000912	We can't just rely on the goddess.{nl}We should do
 QUEST_UNUSED_20151022_000913	We, Kedoran Merchant Alliance helped the goddesses secretly until now.{nl}But, since the world turned into a chaos, we are going to help her actively.
 QUEST_UNUSED_20151022_000914	We are in a trouble..{nl}There are so many things to retrieve, but I don't know what people are doing.
 QUEST_UNUSED_20151022_000915	Ah, so Leopoldas sent you here?{nl}Finally, it seems he has stopped being so stubborn.{nl}
+QUEST_UNUSED_20151022_000916	A long time ago, I was sending the sacred objects to Fedimian.{nl}But, I didn't realise it was the day of Medzio Diena.{nl}
```
Comment (Sunaries) [inline EnglishTranslation-master/QUEST_UNUSED.tsv]: `it was the day of Medzio Diena` -> `it was on Medzio Diena`

Medzio Diena = Day of the Divine Tree

**Comment by Sunaries** [comment]
Good to merge. :+1:

**Comment by imcgames** [comment]
:+1:

**Pull request merged**